modules/ocf_printhost/manifests/enforcer.pp

```diff
@@ -1,6 +1,10 @@
 class ocf_printhost::enforcer {
   package { ['cups-tea4cups', 'mariadb-client']: }
 
+  $mysql_password = assert_type(Pattern[/^[a-zA-Z0-9]*$/], hiera('ocfprinting::mysql::password'))
+
```

Comment:
I'd just remove this newline, both lines I'd say are related (and similar) enough to be together